Datagrid/Datagrid.php

```diff
@@ -126,15 +126,15 @@ public function buildPager()
             $this->formBuilder->add($filter->getFormName(), $type, $options);
         }
 
-        $this->formBuilder->add('_sort_by', 'hidden');
+        $this->formBuilder->add('_sort_by', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
```

Comment:
Stop :) Read the docs: https://sonata-project.org/bundles/core/master/doc/reference/form_types.html#symfony3-supports